core-profile/jsonp-standalone-tck/pom.xml

```diff
@@ -22,11 +22,15 @@
     <artifactId>jsonp-standalone-tck-runner</artifactId>
     <name>WildFly Jakarta JSON-P Standalone TCK Runner</name>
```

Comment:
I think we kind of need a new pluggability tck runner instead of changing the existing standalone tck runner.

Reply:
We can, but it will be a lot of duplication.